odoo/addons/base/__init__.py

```diff
@@ -6,6 +6,10 @@
 from . import report
 from . import wizard
 
+# backward compatibility hook
+# > from odoo.addons import base as dp
+# > dp.get_precision
+from .models.decimal_precision import get_precision
```

Comment:
Do we need this, considering there is a generic compatibility layer now, that handles the old imports correctly?